DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh

```diff
@@ -169,11 +169,14 @@ fi
 
 echo "[INFO (async_pass.sh)] envvars were set to TFDELAYSECONDS ${TFDELAYSECONDS} TIMEFRAME_RATE_LIMIT ${TIMEFRAME_RATE_LIMIT}"
 
+if [[ -z "$SHMSIZE" ]]; then export SHMSIZE=$(( 16 << 30 )); fi
```

Comment:
This will not work on alien, in the sense that there, every env variable is exported as `ALIEN_JDL_XXXYYYZZZ` so here it should be:
```
if [[ -z "$ALIEN_JDL_SHMSIZE ]]; then... 
else
SHMSIZE=$ALIEN_JDL_SHMSIZE
```
Same below.